tests/Feature/StorageProvidersTest.php

```
@@ -57,13 +53,107 @@ public function test_create(array $input): void
+                'state' => 'state-123',
+                'name' => 'dropbox-test',
+                'app_key' => 'my-app-key',
+                'app_secret' => 'my-app-secret',
+                'global' => false,
+            ],
+        ])->get(route('storage-providers.dropbox.callback', ['state' => 'state-123']));
+
+        $response->assertRedirect(route('storage-providers'));
+        $response->assertSessionHas('error');
+        $this->assertDatabaseMissing('storage_providers', ['profile' => 'dropbox-test']);
+    }
```

PHPMD (2.15.0)

[error] The method test_dropbox_callback_rejects_missing_code is not named in camelCase. (undefined)

(CamelCaseMethodName)